apps/studio/components/interfaces/Auth/Users/UsersListItem.tsx

```diff
@@ -32,7 +32,7 @@ const UserListItem = ({
 
   return (
     <Table.tr className="relative" key={user.id}>
-      <Table.td className="whitespace-nowrap">
+      <Table.td className="whitespace-nowrap max-w-xs truncate" title={getDisplayName(user)}>
```

Comment:
Does the table header respects the width of this ? Can't remember how tables work for widths in table row